libvast/test/default_table_slice.cpp

```diff
@@ -63,11 +63,18 @@ struct fixture {
     return builder->finish();
   }
 
-  std::vector<event> subset(size_t from, size_t num) {
-    return {test_events.begin() + from, test_events.begin() + (from + num)};
+  std::vector<value> subset(size_t from, size_t num) {
+    return {test_values.begin() + from, test_values.begin() + (from + num)};
   }
 };
 
+template <class T>
+T unbox(caf::optional<T> x) {
+  if (!x)
+    FAIL("unable to unbox value");
+  return std::move(*x);
+}
+
```

Comment:
I've been seeing multiple implementations of `unbox` showing up lately. Perhaps we should consolidate them?!

Reply:
Yeah, I think this should just be a free function in `caf/test/dsl.hpp`.